working/base-interface-final/feature-specification.md

```diff
@@ -422,9 +471,12 @@ combinations don't make sense:
     `final`.
 *   `base`, `interface`, and `final` all control the same two capabilities so
     are mutually exclusive.
+*   `mixin` as a modifier can obviously only be applied to a class.
```

Comment:
Maybe say: It can be combined with any of the other modifiers.

Reply:
Done.